eva-accession-clustering-automation/config_custom.py

```diff
@@ -0,0 +1,42 @@
+from lxml import etree as et
+import json
+import yaml
+import urllib.request
+
+#TODO: Move methods to pyutils repository
+
+def get_properties_from_xml_file(profile, xml_path):
+    tree = et.parse(xml_path)
+    root = tree.getroot()
+    return get_profile_properties(profile, root)
+
+
+def get_properties_from_xml_string(profile, str):
+    root = et.fromstring(str)
+    return get_profile_properties(profile, root)
+
+
+def get_profile_properties(profile, root):
+    properties = {}
+    for property in root.xpath('//settings/profiles/profile/id[text()="' + profile + '"]/../properties/*'):
+        properties[property.tag] = property.text
+    return properties
+
+
+def get_eva_settings_xml_string(token):
+    url = 'https://api.github.com/repos/EBIvariation/configuration/contents/eva-maven-settings.xml'
+    headers = {'Authorization': 'token ' + token, 'Accept' : 'application/vnd.github.raw' }
+    request = urllib.request.Request(url, None, headers)
```

Comment:
Suggest adding retries for API accesses... Github is notorious for its downtime...